nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/samediff/SameDiffTests.java

```diff
@@ -2977,7 +2976,7 @@ public void testPlaceholderShapeValidation() {
         }
 
         //Also try training:
-        SDVariable sum = sd.math.mergeAdd(ph1, ph2, ph3, ph4);
+        SDVariable sum = sd.math.mergeAdd(new SDVariable[]{ph1, ph2, ph3, ph4});
```

Comment:
mergeAdd should be varargs method (mergeAdd(SDVariable...) not SDVariable[])? If not - fix in code generator